tensorboard/plugins/base_plugin.py

```diff
@@ -273,13 +272,6 @@ def __init__(
               also have been created by the tensorboard_zip_file build rule.
           data_provider: Instance of `tensorboard.data.provider.DataProvider`. May
             be `None` if `flags.generic_data` is set to `"false"`.
-          db_connection_provider: Function taking no arguments that returns a
-              PEP-249 database Connection object, or None if multiplexer should be
-              used instead. The returned value must be closed, and is safe to use in
-              a `with` statement. It is also safe to assume that calling this
-              function is cheap. The returned connection must only be used by a
-              single thread. Things like connection pooling are considered
-              implementation details of the provider.
           db_uri: The string db URI TensorBoard was started with. If this is set,
```

Comment:
Should `db_uri` also be removed here, or is it being used in the manager code somehow?

Reply:
Yep; thanks. Done.